### PR TITLE
Snackbar and navigation

### PR DESCRIPTION
**Pull request by varnita619**
-added toast
- navigation 
Screenshot (49)
Screenshot (50)

### CODE REVIEW

File: Components/toast/toast.html [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+<body>
+
+
+    <div class="container">
+
+        <div class="nav-section">
+
+            <div class="nav--align-start">
+                <h1 class="nav-logo">Pallet UI <sub class=" ui-version nav-logo-sub">v1.0</sub></h1>
+                <!-- <img src="/logo.svg" class="nav-logo" alt=""> -->
```
Comment (romabulani) [inline Components/toast/toast.html:41]: You can remove the comment if its not required

**Review by romabulani — APPROVED** [review]
Rest all looks good, you an merge

**Review by rohit8686 — APPROVED** [review]

File: Components/toast/toast.html (hunk 1 above)
Comment (varnita619) [reply Components/toast/toast.html:41]: Sure

**Review by SJTGSHIVAM — APPROVED** [review]
Looks good to me!

**Pull request merged**